### PR TITLE
Feature arc42 08crosscuttingconcepts

### PR DESCRIPTION
**Pull request by UO258425**

### CODE REVIEW

**Comment by luispc1998** [comment]
LGTM, I would like to investigate more how to handle 'cache', in order to improve performance for the low spec devices, as you stated.

File: src/docs/08_concepts.adoc [hunk 1]
```diff
@@ -49,19 +49,82 @@ on this list.
...
+----
+
+A *route* must contain a name, optionally it can have a description.
+Its composed by at least one waypoint and may or not have a list of comments and media.
+
+The *Waypoint* must have latitude and longitude. They can have as optional properties a name, altitude and a physical address.
+
+A *userComment* must contain the text of the comment, the time it was published and the author, being the last one a IRI to a pod profile shape.
+
+And finally, an *Image* or a *Video* must have an IRI to te resource, another IRI to the pod profile of the author and the time it was published
```
Comment (Angelixus) [inline src/docs/08_concepts.adoc]: Do yo mean URI instead of IRI? please review that if you don´t mind

Comment (UO258425) [reply src/docs/08_concepts.adoc]: I meant IRI. Dont know if its wrong but that is what Labra uses in the shex definitios [here](http://shex.io/)

Comment (Angelixus) [reply src/docs/08_concepts.adoc]: Ok, LGTM

**Comment by Raulpemol** [comment]
LGTM

**Pull request merged**